package/tzdata/package

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env bash
+# Copyright (c) 2020 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+# Inspired by https://github.com/archlinux/svntogit-packages/blob/packages/tzdata/trunk/PKGBUILD
+
+pkgnames=(tzdata)
+pkgdesc="Sources for time zone and daylight saving time data"
+url=https://www.iana.org/time-zones
+pkgver=2020f-1
+timestamp=2020-05-04T06:16Z
+section=utils
+maintainer="Eeems <eeems@eeems.email>"
+license="custom: public domain"
+depends=()
+image=base:v1.2.1
+
+_tzcode=2020f
+_tzdata=2020f
+
+# shellcheck disable=SC2206
+source=(
+    https://www.iana.org/time-zones/repository/releases/tzcode${_tzcode}.tar.gz{,.asc}
+    https://www.iana.org/time-zones/repository/releases/tzdata${_tzdata}.tar.gz{,.asc}
+)
+sha256sums=(
+    'cfeeea2a7745164f64bd9f6d76e47916f4ac820c4434493674adbbd4324329c5'
+    'SKIP'
+    '121131918c3ae6dc5d40f0eb87563a2be920b71a76e2392c09519a5e4a666881'
+    'SKIP'
+)
+
+timezones=(
+    'africa' 'antarctica' 'asia' 'australasia'
+    'europe' 'northamerica' 'southamerica'
+    'etcetera' 'backward' 'factory'
+)
+
+prepare() {
+    sed -i "s:sbin:bin:g" Makefile
+}
+
+build() {
+    # Upgrade gawk to latest to avoid a bug
+    export DEBIAN_FRONTEND=noninteractive
+    apt-get update -y
+    apt-get install -y gawk
+
+    make LFLAGS="${LDFLAGS}"
+}
+
+package() {
+    pushd "${srcdir}"
+    # install tzcode stuff
+    make DESTDIR="${pkgdir}" install
+
+    # install tzdata stuff
+    ./zic -b fat -d "${pkgdir}"/usr/share/zoneinfo "${timezones[@]}"
+    ./zic -b fat -d "${pkgdir}"/usr/share/zoneinfo/posix "${timezones[@]}"
+    ./zic -b fat -d "${pkgdir}"/usr/share/zoneinfo/right -L leapseconds "${timezones[@]}"
+    # This creates the posixrules file. We use New York because POSIX requires the daylight savings time rules to be in accordance with US rules.
+    ./zic -b fat -d "${pkgdir}"/usr/share/zoneinfo -p America/New_York
+    install -m444 -t "${pkgdir}"/usr/share/zoneinfo iso3166.tab zone1970.tab zone.tab # zone.tab is depricated and will go soon
+
+    # install license
+    install -Dm644 LICENSE "${pkgdir}"/opt/usr/share/licenses/tzdata/LICENSE
+
+    popd
+
+    # cleanup
+    rm "${pkgdir}/etc/localtime"
+    rmdir "${pkgdir}/etc"
+    mv "${pkgdir}"/usr/{bin,lib,sbin,share/man} "${pkgdir}"/opt/usr
```

Comment:
`/opt/usr/bin` is still no in our path at the moment. (We should really fix that.)

Reply:
I'll just move them to `/opt/bin` for now